pyvista/plotting/plotting.py

```diff
@@ -3282,20 +3285,20 @@ def _save_image(image, filename, return_img=None):
         if not image.size:
             raise ValueError('Empty image. Have you run plot() first?')
         # write screenshot to file
-        supported_formats = [".png", ".jpeg", ".jpg", ".bmp", ".tif", ".tiff"]
         if isinstance(filename, (str, pathlib.Path)):
+            from PIL import Image
```

Comment:
Need to add `pillow` to `setup.py`?

Reply:
On a fresh virtualenv on python3.7:
```
Successfully installed appdirs-1.4.4 imageio-2.9.0 importlib-metadata-2.0.0 meshio-4.3.1 numpy-1.19.2 pillow-8.0.0 pyvista-0.26.0 scooby-0.5.6 vtk-9.0.1 zipp-3.3.1
```
Seems that `imageio` includes `pillow` as a dependency, and while I don't think this will change, it's probably not a bad idea to add direct dependencies to our setup.py

Reply:
Indeed it's always best to specify directly the dependencies that you need (and not rely on what those dependencies do or do not bring in with them). It's both more stable and immediately clear what's used in the codebase. At some point `imageio` could be removed -- for example if we no longer need it, or if there is a better library we want to change to using for whatever it is we still use it for (making movies?), and it would also break in that case...